app/src/main/java/com/example/iou/map/activities/GoogleMapsActivity.java

```diff
@@ -127,7 +132,9 @@ public void onFailure(int statusCode, okhttp3.Headers headers, String response,
     }
 
     // Generates the URL for the API request based on the user's location
-    private String findUsersLocation(Location location) {
+    private String findUsersLocation(Location location,String whereTo) {
```

Comment:
may want to rename this function, now that it also takes in the "whereTo" string. In fact, it might be better to name it something like "generateURL" or "constructMapsURL," since the URL generation is the important part.